projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid-expanding.spec.ts

```diff
@@ -214,6 +214,21 @@ describe('IgxTreeGrid - Expanding / Collapsing', () => {
             expect(rows.length).toBe(4, 'root level row collapsing problem');
         });
 
+        it('should expand/collapse when using \'expandAll\' and \'collapseAll\' methods', () => {
+            treeGrid.perPage = 50;
+
+            let rows = TreeGridFunctions.getAllRows(fix);
+            expect(rows.length).toBe(4);
+
+            treeGrid.expandAll();
+            rows = TreeGridFunctions.getAllRows(fix);
+            expect(rows.length).toBe(12);
+
+            treeGrid.collapseAll();
+            rows = TreeGridFunctions.getAllRows(fix);
+            expect(rows.length).toBe(4);
+        });
+
         it('should emits an event when expanding rows (API)', (done) => {
```

Comment:
All "should emits" occurrences should be "should emit".

Reply:
Done